drivers/adodb-postgres64.inc.php

```diff
@@ -91,6 +91,15 @@ class ADODB_postgres64 extends ADOConnection{
 	var $version;
 	var $_nestedSQL = false;
 
+	/** @var array an array with client, protocol and server values. */
+	var $version;
+
+	/** @var bool */
+	var $_nestedSQL = false;
```

Comment:
duplicate, declared on line 92

Reply:
Hi @matt-land. I just updated it to remove those lines. Thanks.